.eslintrc.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Not sure about this file being included here... don't see any other good solution for now, but it is definitely not our own - comes from Drupal `core/.eslintrc.json` - probably we can get it somehow downloaded... but we want to run the code check without installing (before running `make install` that downloads all the dependencies including this file) - something to think about.